chain/jsonrpc-primitives/src/types/transactions.rs

```diff
@@ -5,17 +5,19 @@ pub struct RpcBroadcastTransactionRequest {
     pub signed_transaction: near_primitives::transaction::SignedTransaction,
 }
 
-#[derive(Debug)]
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
 pub struct RpcTransactionStatusCommonRequest {
+    #[serde(flatten)]
     pub transaction_info: TransactionInfo,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[serde(untagged)]
 pub enum TransactionInfo {
     Transaction(near_primitives::transaction::SignedTransaction),
     TransactionId {
-        hash: near_primitives::hash::CryptoHash,
-        account_id: near_primitives::types::AccountId,
+        tx_hash: near_primitives::hash::CryptoHash,
+        sender_account_id: near_primitives::types::AccountId,
```

Comment:
This seems a bit sus, is it backwards compatible and will any clients break if those names change?

Reply:
We don't provide the option to use this naming externally. The only option now is to pass the arguments as the vector (without field names, the order should be taken from the documentation).
I want to add the option to pass object (with field names) as params, and I think it's better to make names self-explanatory before this change.

If I change the naming at response - yes, it would not be safe. Since it's only request, we should be fine.

Reply:
Great, thanks for clarifying that up for me :) 
+1 for better naming, good idea!